packages/io-wallet-common/src/azure-health-check.ts

```diff
@@ -22,3 +23,20 @@ export const getCosmosHealth: RTE.ReaderTaskEither<
     ),
     TE.map(() => true),
   );
+
+export const getStorageQueueHealth: RTE.ReaderTaskEither<
+  {
+    queueClient: QueueClient;
+  },
+  Error,
+  true
+> = ({ queueClient }) =>
+  pipe(
+    TE.tryCatch(
+      () => queueClient.exists(),
+      () => new Error("storage-queue-error"),
```

Comment:
```suggestion
      E.toError,
```

Reply:
all health check functions return the generic error message of the resource that generated the error